### PR TITLE
Additional state for VBA Document Upload

### PR DESCRIPTION
**Pull request by ATeal**
## Description of change
support vbms success, but only keep the states we currently have

resolves https://github.com/department-of-veterans-affairs/vets-contrib/issues/1567

## Testing done
- rspec

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: modules/vba_documents/README.yml [hunk 1]
```diff
@@ -387,6 +387,7 @@ components:
             * `received` - Indicates document package has been successfully propagated to Central Mail API
             * `processing` - Indicates document package is being processed by ICMHS or a downstream system.
             * `success` - Indicates document package has been received by DHMS.
+            * `vbms success` - Indicates that the the package has been succesfully processed into VBMS.
```
Comment (bastosmichael) [inline modules/vba_documents/README.yml]: All of the statuses are singularly worded, should we consider vbms_success possibly?

**Comment by zachgoldfine44** [comment]
@AubreySuter you may want to update the Benefits Intake API one-pager and any other supporting documentation we use to share with stakeholders. You don't need to add it to this PR but just a thought.

**Comment by AubreySuter** [comment]
Great point @zachgoldfine44. I'll make a task to do that and assign it to myself.

File: modules/vba_documents/README.yml (hunk 1 above)
Comment (bastosmichael) [reply modules/vba_documents/README.yml]: Remove

File: modules/vba_documents/app/models/vba_documents/upload_submission.rb [hunk 2]
```diff
@@ -107,6 +107,8 @@ def map_downstream_status(response_object)
         self.status = 'processing'
       when 'Success'
         self.status = 'success'
+      when 'VBMS Success'
+        self.status = 'vbms success'
```
Comment (bastosmichael) [inline modules/vba_documents/app/models/vba_documents/upload_submission.rb]: just default to success

**Pull request merged**